src/streamlink/plugins/huomao.py

```diff
@@ -1,23 +1,23 @@
 # NOTE: Since a documented API is nowhere to be found for Huomao; this plugin
```

Comment:
Python has a block comment syntax (also it's a string)
```python
"""
A long,
   multi-line
comment
"""
```
If they are at the top of a module, they because the `docstring` for module/method (see [PEP-257](https://www.python.org/dev/peps/pep-0257/) for more info on the subject).